### DISCUSSION TITLE
saving issue

### DISCUSSION THREAD

**Discussion by ramirss** [OnionUI/Onion#1646] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3

### Miyoo Firmware Version

202306282128

### Provide a clear and concise description of the issue

I cannot save any game anymore 
whatever i save in a reboot all will be deleted and it's so frustrating.

### Steps to Reproduce

maybe after the update with the OTA because it happened when i updated from 4.2 to 4.3

### Provide any additional context or information that may help us investigate your issue

i searched for others issue but non of the waysworked

**Comment by XDarkBringX** [comment]
is this the stock sd card? i find plenty of posts that this was the main issues with saves not staying. another issues could be a different core has been selected but i doubt this is the case.

**Comment by ramirss** [comment]
no it's not the stock sd card.
